src/monitor.rs

```diff
@@ -50,17 +50,16 @@ impl Iterator for AvailableMonitorsIter {
 /// - [`MonitorHandle::video_modes`][monitor_get].
 ///
 /// [monitor_get]: ../monitor/struct.MonitorHandle.html#method.video_modes
-#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+#[derive(Derivative)]
+#[derivative(Clone, Debug = "transparent")]
 pub struct VideoMode {
```

Comment:
I realize this specific feedback would've been better-placed in the original PR, but it's only occurred to me now: is it possible to add a function to `Monitor` to retrieve the currently-active video mode? That way, applications can launch in fullscreen with the sensible default of *whatever fullscreen settings the monitor currently has*, instead of having to guess which mode to use.

Reply:
Shouldn't be a problem, but I'm inclined to do this in a separate pull request since I will have to implement it for all four platforms. I think the borderless mode would suffice for a sensible default in any case.